types/wlr_output.c

```diff
@@ -958,6 +961,9 @@ void wlr_output_cursor_set_surface(struct wlr_output_cursor *cursor,
 
 	output_cursor_reset(cursor);
 
+	if (cursor->visible && cursor->surface) {
```

Comment:
This should already be taken care of by the previous call to `output_cursor_reset`, or am I missing something?

Reply:
It is, fixed.